### PR TITLE
Ensure categories and tags mapping is applied

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

**Review by tpage-alfresco — CHANGES_REQUESTED** [review]
Ah - sorry, would it be possible to make this change in the Create test instead? This property is already being used to test the weird `null` to `null` case.

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java [hunk 1]
```diff
@@ -124,4 +124,244 @@ void testUpdateRequest()
...
+
+        // when
+        containerSupport.raiseRepoEvent(repoEvent);
+
+        // then
+        String expectedBody = """
+                {
+                  "objectId" : "d71dd823-82c7-477c-8490-04cb0e826e65",
+                  "eventType" : "update"
+                }""";
```
Comment (tpage-alfresco) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java:229]: Hmm - this is a weird edge case. I guess really we shouldn't send anything in this case, but actually it should never happen since I think the modification date will always be updated.

Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java:229]: I can actually add some random property like title updated wdyt?

Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java:229]: so we'll have 
image

**Review by tpage-alfresco — APPROVED** [review]
Nice collection of test cases :+1:

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java (hunk 1 above)
Comment (tpage-alfresco) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java:229]: I'm happy either way. Like this we at least check what happens when there's no real change, even if it's a bit weird to tell Hx Insight "there was no change to this node".
